packages/calling-component-bindings/src/handlers/createHandlers.ts

```diff
@@ -220,7 +223,7 @@ export const createDefaultCallingHandlers = memoizeOne(
       }
 
       if (screenShareStream && screenShareStream.isAvailable && !screenShareStream.view) {
-        callClient.createView(call.id, participant.identifier, screenShareStream, options);
+        callClient.createView(call.id, participant.identifier, screenShareStream, { scalingMode: 'Fit' });
```

Comment:
hardcoded this here since it is highly unlikely that CONTOSO would ever want to use a different scaling mode for screenshare.
Using `Crop` would crop the contents of screenshare and `Stretch` would warp it.
`Fit` is the only mode that maintains the integrity of the screen being shared.

Reply:
Please add this comment to the code for posterity.